.github/workflows/auto-update-gemfiles.yml

```diff
@@ -0,0 +1,169 @@
+name: Auto-update Gemfiles
+
+# This action cannot be skipped altogether because it is a mandatory status check.
+# Instead we conditionally skip it at job level, instead of workflow level.
+on:
+  schedule:
+    - cron: '0 0 * * 0' # Every Sunday at midnight
```

Comment:
Just want to note to the other Ruby people here that this automation will start running